sphinxcontrib/cacoo.py

```diff
@@ -1,7 +1,9 @@
 import re
 import json
 from time import mktime
-from urllib2 import urlopen
+from future.standard_library import install_aliases
```

Comment:
`future` is not a part of dependencies. So it would get failed. Could you use `six` package instead?
You can use it without change dependencies because Sphinx already uses it.

Reply:
Thank you for your comment. I updated to use `six` instead of `future`.